recipe/recipe.yaml

```diff
@@ -39,19 +41,19 @@ requirements:
   run:
     - python
 
-test:
-  imports:
-    - rustac
-  commands:
-    - pip check
-    - rustac --help
-  requires:
-    - pip
+tests:
+  - python:
+      imports:
+        - rustac
+      pip_check: true
+  - script:
+      - rustac --help
 
 about:
-  home: https://stac-utils.github.io/rustac
+  homepage: https://stac-utils.github.io/rustac
+  repository: https://github.com/stac-utils/rustac
   summary: A no-dependency Python package for STAC, using Rust under the hood.
-  license: MIT AND Apache-2.0
+  license: MIT OR Apache-2.0
```

Comment:
Should be `MIT OR Apache-2.0` according to https://github.com/rust-lang/cargo/issues/2039 and https://wiki.spdx.org/view/SPDX_FAQ#Using_the_SPDX_Spec (search for dual license).